src/contribute/contribution-guide.md

```diff
@@ -304,6 +304,18 @@ At this point, you want to ensure everything is right. Test it with `mvn verify`
 
 Done. You can delete the local <code>finish-pr-<pull-request-#></code> branch if you like.
 
+## Granting more rights to a contributor
+
+The project management committee (PMC) can grant more rights to a contributor, such as commit access or decision power, and recognize them as new committers or PMC members.
+
+The PMC periodically discusses this topic and privately votes to grant more rights to a contributor. If the vote passess, the contributor is invited to accept or reject the nomination. Once accepted, the PMC announces the decision publicly and updates the list of team member accordingly.
+
+The key to the selection process is [Meritocracy](http://apache.org/foundation/how-it-works.html#meritocracy), literary government by merit. Contributors can get merit in many ways: contributing code, testing releases, participating in documentation effort, answering user questions, debating design proposals, triaging issues, evangelizing the project, growing user base, and any other action that benefits the project as a whole.
+
+Therefore, there isn’t a single committer bar, e.g., a specific number of commits. In most cases, new committers will have a combination of different types of contributions that are hard to compare to each other. Similarly, there isn’t a single PMC bar either. The PMC discusses all contributions from an individual, and attempts to apply the same standard to all contributors.
```

Comment:
'attempts to apply the same standard' is odd after saying there isn't a single standard. Perhaps 'evaluates the overall impact across all the above dimensions'?